api/odigos/v1alpha1/instrumentatioconfig_types.go

```diff
@@ -41,16 +42,76 @@ type SdkConfig struct {
 
 	// configurations for the instrumentation libraries the the SDK should use
 	InstrumentationLibraryConfigs []InstrumentationLibraryConfig `json:"instrumentationLibraryConfigs"`
+
+	// HeadSamplingConfig is a set sampling rules.
+	// This config currently only applies to root spans.
+	// In the Future we might add another level of configuration base on the parent span (ParentBased Sampling)
+	HeadSamplingConfig HeadSamplingConfig `json:"headSamplerConfig,omitempty"`
+}
+
+// 'Operand' represents the attributes and values that an operator acts upon in an expression
+type AttributeCondition struct {
+	// attribute key (e.g. "url.path")
+	Key attribute.Key `json:"key"`
+	// currently only string values are supported.
+	Val string `json:"val"`
+	// The operator to use to compare the attribute value.
+	Operator Operator `json:"operator"`
```

Comment:
Maybe we should have `omitempty` here for the operator. This will result in the default operation being `equals` if we define a default value for the `Operator` as suggested in the other comment